cpu/esp32/esp_xtimer.c

```diff
@@ -118,10 +117,18 @@ void ets_timer_setfn(ETSTimer *ptimer, ETSTimerFunc *pfunc, void *parg)
 
     CHECK_PARAM(e2xt != NULL);
 
+    /* cppcheck-suppress nullPointerRedundantCheck
+     * CHECK_PARAM() sends cppcheck off rails */
```

Comment:
Would an `assert` make cpp-check happy here?

Reply:
Let's give it a try :-)

Reply:
The `assert()`s do work.